src/less/components/form.less

```diff
@@ -168,7 +154,7 @@ legend {
 .form-control {
     display: block;
     width: 100%;
-    font-size: @base-font-size;
+    //font-size: @base-font-size;
```

Comment:
why comment?
let's remove then?
or you mean since it's base-font-size then let's use `1rem`? (which I do agree with, I do not understand the point of `@base-font-size` VS using `rem` 🤔 )